docker-compose.yml

```diff
@@ -5,7 +5,6 @@ services:
     build: ./docker/wordpress_xdebug
     image: wordpress-xdebug
     container_name: woocommerce_stripe_wordpress
-    restart: always
```

Comment:
Quick question, why are we removing the restart policy?

Reply:
For consistency, only the WordPress container was started automatically after a reboot.
Another option would've been adding always to the DB container, but we don't usually want the containers to start after every reboot.